src/main/java/hudson/plugins/jira/Updater.java

```diff
@@ -341,4 +346,24 @@ static void findIssues(AbstractBuild<?, ?> build, Set<String> ids, Pattern patte
      * Debug flag.
      */
     public static boolean debug = false;
+
+    public static final class DefaultUpdaterIssuesSelector extends UpdaterIssuesSelector {
```

Comment:
Please move to a separate class file and check if that is covered by the existing unit tests. 
I guess it will be, but if not, please create a new one.